packages/apps/webhooks/src/events-to-webhooks-app.ts

```diff
@@ -0,0 +1,178 @@
+import type { ApiEventHandler } from '@cloudcommerce/firebase/lib/helpers/pubsub';
+import type { Carts, Customers, Orders } from '@cloudcommerce/api/types';
+import logger from 'firebase-functions/logger';
+import api from '@cloudcommerce/api';
+import axios from 'axios';
+
+const updateManualQueue = async (
+  manualQueue: any[] | null,
+  appId: string,
+) => {
+  if (manualQueue) {
+    try {
+      await api.patch(`applications/${appId}`, { data: { manual_queue: manualQueue } });
+      manualQueue = null;
+    } catch (err) {
+      logger.error(err);
+    }
+  }
+};
+
+const sendWebhook = async (
+  options: { [x: string]: any },
+  doc: Record<string, any>,
+  urls: any[],
+  manualQueue: any[] | null,
+  appId: string,
+  customer?: Customers,
+  docId?: string,
+  isCart?: boolean,
+) => {
+  const url = options && options.webhook_uri;
+  if (url && !urls.includes(url) && (!isCart || options.send_carts)) {
+    urls.push(url);
+    logger.log(`Event for Store ${isCart ? 'cart' : 'order'} => ${url}`);
```

Comment:
```suggestion
    logger.log(`Event ${isCart ? 'cart' : 'order'} => ${url}`);
```

Só tem uma loja por deploy 😄